Source/Editor/DualityEditor/Controls/AutoExpandTreeView.cs

```diff
@@ -0,0 +1,68 @@
+using Aga.Controls.Tree;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Duality.Editor.Controls
+{
+	public class AutoExpandTreeView : TreeViewAdv
+	{
+		public int DragOverExpandDelay
+		{
+			get;
+			set;
+		} = 1000;
+
+		private TreeNodeAdv autoExpandNode = null;
+
+		private Timer timer = new Timer();
+
+		private void StartTimer()
+		{
+			if (DragOverExpandDelay > 0)
+			{
+				timer.Stop();
+				timer.Interval = DragOverExpandDelay;
+				timer.Tick += OnTimerTick;
```

Comment:
Yep, we were re-listening every time, possibly degrading performance over time. Now just doing it in a constructor and unsubscribing in `Dispose`. Also added an optional parameter for the constructor to take in the ms for the `autoExpandDelayTimer` interval (1000 is default).